web/src/features/sfmsInsights/components/map/sfmsFeatureStylers.ts

```diff
@@ -38,3 +41,15 @@ export const styleFuelGrid = () => {
   }
   return style
 }
+
+// A styling function for the snow coverage pmtiles layer.
+export const snowStyler = (feature: RenderFeature | ol.Feature<Geometry>): Style => {
```

Comment:
Could add a couple tests here

Reply:
Good idea. I added tests for the new `snowStyler` and the existing `styleFuelGrid` function.